lib/topological_inventory/orchestrator/source_type.rb

```diff
@@ -40,6 +40,12 @@ def supported_source_type?
       def supports_availability_check?
         AVAILABILITY_CHECK_SOURCE_TYPES.include?(attributes['name'])
       end
+
+      private
+
+      def enabled_source_types
+        @enabled_source_types ||= ENV['ENABLED_SOURCE_TYPES']&.split(",") || []
```

Comment:
@lindgrenj6 Please move ENV variable to entrypoint https://github.com/RedHatInsights/topological_inventory-orchestrator/blob/master/bin/topological_inventory-orchestrator. 
With default values equal to current source types

Reply:
btw env var can be nil, so `split` will fail in this case

Reply:
> Please move ENV variable to entrypoint /bin/topological_inventory-orchestrator@master .
With default values equal to current source types

I'm not 100% sure how you wanted me to do this - but I basically added it as an argument to optimist, then as another argument to the Worker initializer with a default value of the original values.

> btw env var can be nil, so split will fail in this case

Yep, can't risk the NoMethodError, that's why I used the safe operator `&.` which will just return nil if the `ENV` var is nil.

Reply:
I have some spec failures I'm having trouble with, it's just the `targeted_update_spec.rb` file, is there something I'm missing there?

Reply:
> that's why I used the safe operator &.

Sure, I was thinking about something else :D